sdk/test-utils/recorder/src/utils/requestBodyTransform.ts

```diff
@@ -22,15 +22,18 @@ export const defaultRequestBodyTransforms: RequestBodyTransformsType = {
     //    and
     // 2. as a filter on the new requests to be able to match the request bodies
     (body: string) =>
-      body.replace(/client-request-id=[^&]*/g, "client-request-id=client-request-id")
+      body.replace(/client-request-id=[^&]*/g, "client-request-id=client-request-id"),
+    // Sanitizes the scope values in the recordings - to reduce the noise from cred scan reports
+    (body: string) =>
+      body.replace(/scope=https%3A%2F%2F(.+?)(&|")/g, "scope=https%3A%2F%2Fsanitized%2F$2")
```

Comment:
I later found out that the regex should be 
```suggestion
      body.replace(/scope=https%3A%2F%2F(.+?)(&+|$)/g, "scope=https%3A%2F%2Fsanitized%2F$2")
```

Reply:
![image](https://user-images.githubusercontent.com/10452642/115483503-c47d1500-a205-11eb-9a7a-704a98de1ecc.png)
$2 wasn't working somehow

Reply:
There was some problem with $2. Switched to `[^&"]*` instead.